go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/iancoleman/strcase v0.3.0
 	github.com/ionos-cloud/sdk-go-api-gateway v1.0.0
 	github.com/ionos-cloud/sdk-go-bundle/products/cdn/v2 v2.1.0
+	github.com/ionos-cloud/sdk-go-bundle/products/containerregistry/v2 v2.0.2
```

Comment:
I think you forgot to remove the old entry
	github.com/ionos-cloud/sdk-go-container-registry v1.2.0